src/deserialize/parse.rs

```diff
@@ -54,6 +52,23 @@ fn read_str(chars: &mut std::str::Chars) -> Edn {
     Edn::Str(string)
 }
 
+fn read_others(a: char, chars: &mut std::str::Chars) -> Result<Edn, Error> {
```

Comment:
I don't know if I like the `others` suffix :thinking:

Reply:
c49169c